README.md

```diff
@@ -40,3 +40,46 @@ This repo contains a UML sequence diagram, which can be updated as follows:
 1. Run `docker pull think/plantuml` to download [this useful PlantUML image](https://hub.docker.com/r/think/plantuml/)
 2. Edit `./sequenceDiagram/source.uml` as needed
 3. Run `cat sequenceDiagram/source.uml | docker run --rm -i think/plantuml -tpng > sequenceDiagram/source.png` to output
+
+## Workflow
+
+The process of converting and sending a document is handled by a number of interrelated services:
+
+### Fetch task (run once a minute)
+
+- Fetch all new documents in the print queue from comino (compare with latest document in Dynamo)
+- Store in Dynamo (state = `Waiting`)
+- Push into SQS
+
+### Conversion task
+
+- Fetch document ID from SQS
```

Comment:
The lambda function gets triggered with the ID from the queue as opposed to fetching it. I'm not sure if it's worth making that clear.